_build_cfg.yml

```diff
@@ -363,6 +321,58 @@ Topics:
     Distros: openshift-enterprise
 
 ---
+Name: Storage Configuration
+Dir: storage_config
+Distros: openshift-origin,openshift-enterprise
```

Comment:
@aweiteka The persistent storage topics were not previously in the Atomic Registry docs (see Distros on line 297). Just want to make sure you don't want them in AR docs still, now that they'll be in a separate storage config guide.